test/HarpoonBattleTest.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ * File:   HarpoonBattleTest.cpp
+ * Author: Kjell Hedstrom
+ *
+ * Created on November 30, 2015
+ */
+
+
+#include "HarpoonBattleTest.h"
+#include "Kraken.h"
+#include "KrakenBattle.h"
+#include "HarpoonBattle.h"
+#include "KrakenIntegrationHelper.h"
+#include <Death.h>
+#include <memory>
+#include <tuple>
+#include <utility>
+
+
+TEST_F(HarpoonBattleTest, EnumToString) {
+   auto data = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::Data);
+   auto done = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::Done);
+   auto error = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::Error);
+   auto end = HarpoonBattle::EnumToString(HarpoonBattle::ReceivedType::End);
+
+   EXPECT_TRUE(data == "<DATA>");
+   EXPECT_TRUE(done == "<DONE>");
+   EXPECT_TRUE(error == "<ERROR>");
+   EXPECT_TRUE(end == "<END>");
+
+}
+
+TEST_F(HarpoonBattleTest, EnumToString_Converted) {
```

Comment:
The test verifies that the enum from HarpoonBattle equals the corresponding enum from KrakenBattle

I can change according to name conventions